### PR TITLE
vmtouch -> 1.0.2

### PR DESCRIPTION
**Pull request by garrison**

### CODE REVIEW

**Comment by pSub** [comment]
Looks good, but could you Indicate in your commit message that vmtouch is upgraded vom the git revision to 1.0.2?

File: pkgs/tools/misc/vmtouch/default.nix [hunk 1]
```diff
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "vmtouch";
-  version = "git-20150310";
+  version = "1.0.2";
```
Comment (jagajaga) [inline pkgs/tools/misc/vmtouch/default.nix]: Unfortunately the `git` part of the name is the name. So now if you remove `git` anyone who had installed this tool by not attribute path will miss the update..

Comment (garrison) [inline pkgs/tools/misc/vmtouch/default.nix]: I realized this too when I did the upgrade.  It seems that it is the convention to put `-git` as part of the name for packages that come from git (there are numerous examples of this, and is why I did it originally).  Given the circumstances, is there a better solution than what I've done with this upgrade?

Comment (jagajaga) [inline pkgs/tools/misc/vmtouch/default.nix]: Just write `git-1.0.2` :)

Comment (garrison) [inline pkgs/tools/misc/vmtouch/default.nix]: I can do that if that's the consensus, but it seems strange to me to keep `git-` in the name indefinitely if it's using a released version from now on.

Comment (jagajaga) [inline pkgs/tools/misc/vmtouch/default.nix]: I agree with you, that it is strange, but that's the only way to keep backwards compatibility. Or wait for nix to have feature to parse `git` part..

Comment (globin) [inline pkgs/tools/misc/vmtouch/default.nix]: I'd personally drop it anyway because I think this will create more confusion..

Comment (jagajaga) [inline pkgs/tools/misc/vmtouch/default.nix]: @globin I wrote why do we need to do it like that.

Comment (globin) [inline pkgs/tools/misc/vmtouch/default.nix]: I know but I'd still prefer dropping the suffix nonetheless.

Comment (jagajaga) [inline pkgs/tools/misc/vmtouch/default.nix]: Anyway we won't do this.

Comment (jagajaga) [inline pkgs/tools/misc/vmtouch/default.nix]: We don't want users to run into situation of outdated pkgs. I hope I will patch nix to "eat" such things in futute, so we will remove this part in version.

**Comment by garrison** [comment]
Commit message updated, should be good to be applied now.

**Comment by jagajaga** [comment]
@garrison but what about `git` suffix? Are we dropping users?

**Comment by jgillich** [comment]
Breaking backwards compatibility sucks, but I think it's the right thing to do here. People that install packages with `git-` in their names surely are aware that it will go away once actual releases are made?

**Comment by jagajaga** [comment]
@jgillich I don't know :)
But in future that won't matter if we patch `nix`.

**Comment by garrison** [comment]
OK, I changed it to keep `git-` in the version, as you insist.  Should be good to go now.

**Comment by garrison** [comment]
Thanks!

**Comment by garrison** [comment]
Unfortunately, I fear that now people are being left behind anyway, as the version resolver considers the old `git-20150310` to be an upgrade over the newer package, `1.0.2`.

```
$ nix-env -u --dry-run
(dry run; not doing anything)
upgrading ‘vmtouch-git-1.0.2’ to ‘vmtouch-git-20150310’
these paths will be fetched (0.01 MiB download, 0.03 MiB unpacked):
  /nix/store/kddr5cfxasph75s1a1w9hhc7acdkb85h-vmtouch-git-20150310
```

**Comment by garrison** [comment]
> Unfortunately, I fear that now people are being left behind anyway, as the version resolver considers the old git-20150310 to be an upgrade over the newer package, 1.0.2.

@jagajaga Any thoughts on this?  I am tempted to make another pull request, removing the `git-` from the version since people are being left behind anyway.

**Comment by jagajaga** [comment]
Well, you can set date of release as a version.

**Pull request merged**